transformer_engine/pytorch/float8_tensor.py

```diff
@@ -206,6 +206,87 @@ def forward(
     def backward(ctx, grad):
         return grad.to(ctx.input_dtype), None
 
+class _ViewFunc(torch.autograd.Function):
```

Comment:
@timmoon10 Do we prefer adding a `_ViewFunc` or registering `aten.view.default` in `__torch_dispatch__`?

Reply:
`view()` and `reshape()` here produce `Float8Tensor`s rather than `torch.Tensor`s, and it has proven helpful when passing `Float8Tensor`s around. Let me know if there's a better way to do it though.